stock_picking_batch_back2draft/models/stock_picking_batch.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Studio73 - Guillermo Llinares <guillermo@studio73.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class StockPickingBatch(models.Model):
+    _inherit = "stock.picking.batch"
+
+    def action_back_to_draft(self):
+        picking_ids = self.mapped("picking_ids")
```

Comment:
IMHO you should filtered on pickings that have "cancel" state here.

Reply:
Agreed, this way if there are pickings in different states, the cancelled ones will be reset instead of getting a warning saying that all of them have to be cancelled.